src/view-browser.html

```diff
@@ -142,6 +142,7 @@ <h1 id='sidebar-title' class='sidebar-title'></h1>
 <script type='text/javascript' src='mxnet.js'></script>
 <script type='text/javascript' src='cntk.js'></script>
 <script type='text/javascript' src='sklearn.js'></script>
+<script type='text/javascript' src='openvino-ir.js'></script>
```

Comment:
`openvino.js`

Reply:
renamed

is the discussion resolved now?